package.json

```diff
@@ -66,7 +66,7 @@
     "uiFramework:build": "grunt uiFramework:build",
     "uiFramework:createComponent": "yo ./ui_framework/generator-kui/app/component.js",
     "uiFramework:documentComponent": "yo ./ui_framework/generator-kui/app/documentation.js",
-    "ts:build": "tsc",
+    "ts:build": "tsc -p platform/tsconfig.json",
```

Comment:
__optional nit:__ IIRC `tsconfig.json` is optional, so just `tsc -p platform` should work too.

Reply:
:+1: ✅